modules/framework/galasa-parent/dev.galasa.framework.api.cps/src/main/java/dev/galasa/framework/api/cps/internal/CpsServlet.java

```diff
@@ -38,6 +42,8 @@ public class CpsServlet extends BaseServlet {
 	private static final long serialVersionUID = 1L;
 
 	private Log  logger  =  LogFactory.getLog(this.getClass());
+
+	protected Environment env = new SystemEnvironment();
```

Comment:
Might be worth having 2 constructors, one where this is passed in, and the no-parameter one calls the parameterized one.